ui/src/app/question/basequestion/question-body.component.html

```diff
@@ -7,72 +7,74 @@
     }
 </style-->
 <div ngModelGroup="questionBody" id="questionBody" class="exclude">
-    @if (showWarning()) {
-        <div class="xm-study-item-container--warning mt-3">
-            <!-- Exam contexts -->
-            <div class="row">
-                <div class="col-auto p-1">
-                    <img src="/assets/images/icon_warning.png" />
-                </div>
-                <div class="col warning-text">
-                    @if (examNames.length > 1) {
-                        <div class="col warning-text">
-                            {{ 'i18n_exam_question_edit_instructions' | translate }}
-                        </div>
-                    }
-                    @if (examNames.length === 1) {
-                        <div class="col warning-text">
-                            {{ 'i18n_exam_question_edit_instructions_for_one' | translate }}
-                        </div>
-                    }
-                    @if (examNames.length <= 0) {
-                        <div class="col warning-text">
-                            {{ 'i18n_exam_question_edit_instructions_for_none' | translate }}
-                        </div>
-                    }
-                    <div class="col-12 justify-content-center m-4">
-                        <ul class="list-inline row">
-                            <li class="list-inline-item relation-text mb-3 col-12">
+    <div class="xm-study-item-container--warning mt-3">
+        <!-- Exam contexts -->
+        <div class="row">
+            <div class="col-auto p-1">
+                <img src="/assets/images/icon_warning.png" />
+            </div>
+            <div class="col warning-text">
+                @if (examNames.length > 1) {
+                    <div class="col warning-text">
+                        {{ 'i18n_exam_question_edit_instructions' | translate }}
+                    </div>
+                }
+                @if (examNames.length === 1) {
+                    <div class="col warning-text">
+                        {{ 'i18n_exam_question_edit_instructions_for_one' | translate }}
+                    </div>
+                }
+                @if (examNames.length < 1) {
+                    <div class="col warning-text">
+                        {{ 'i18n_exam_question_edit_instructions_for_none' | translate }}
+                    </div>
+                }
+                <div class="col-12 justify-content-center m-4">
+                    <ul class="list-inline row">
+                        <li class="list-inline-item relation-text mb-3 col-12">
+                            <span class="text-nowrap">
+                                {{ 'i18n_exam_question_in_use' | translate }}: {{ examNames.length }}
+                            </span>
+                        </li>
+
+                        @for (name of sortByString(examNames).slice(0, 5); track name) {
```

Comment:
Eikö tässä voisi olla ihan `name of examNames.sort().slice(0, 5)` niin ei tarttis tuollaista funktiota edes tehdä?

Reply:
Katoin just samaa.